cmd/thanos/query.go

```diff
@@ -82,9 +82,13 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 	fileSDInterval := modelDuration(cmd.Flag("store.sd-interval", "Refresh interval to re-read file SD files. It is used as a resync fallback.").
 		Default("5m"))
 
+	// TODO(bwplotka): Grab this from TTL at some point.
 	dnsSDInterval := modelDuration(cmd.Flag("store.sd-dns-interval", "Interval between DNS resolutions.").
 		Default("30s"))
 
+	dnsSDResolver := cmd.Flag("store.sd-dns-resolver", fmt.Sprintf("Resolver to use. Possible options: [%s, %s]", dns.GolangResolverType, dns.MiekgdnsResolverType)).
```

Comment:
can you link to the issue so that people know why they would use one over the other? Should this one also be hidden like in ruler?

Reply:
I think we should unhide maybe (or hide) in all of them, my mistake. For reference.. not sure. Do you think that\s the proper place for it?

Reply:
if we don't hide the flag, I think this is the place where it will be nice to see what they actually do. As a user I have no idea what miekgdns is.